.gas-snapshot

```diff
@@ -469,18 +469,22 @@ ProductIdUtilTest:testFormatAndParseAreMirrored(uint8,uint8,uint8,uint8,uint8) (
 TokenIdUtilTest:testFormatAndParseAreMirrored(uint8,uint40,uint64,uint64,uint64) (runs: 256, μ: 4599, ~: 4599)
 TokenIdUtilTest:testGetAndParseAreMirrored(uint8,uint40,uint256,uint256,uint256) (runs: 256, μ: 4354, ~: 4354)
 TokenIdUtilTest:testTokenIdHigherThan0(uint8,uint40,uint64,uint64,uint64) (runs: 256, μ: 4203, ~: 4203)
-ArrayUtilTest:testArgSort() (gas: 8966)
-ArrayUtilTest:testArgSortDups() (gas: 8919)
-ArrayUtilTest:testArgSortDupsEvenItems() (gas: 11028)
-ArrayUtilTest:testArgSortDupsInt() (gas: 8963)
-ArrayUtilTest:testConcat() (gas: 5152)
+ArrayUtilTest:testArgSort() (gas: 13937)
+ArrayUtilTest:testArgSortDups() (gas: 13504)
+ArrayUtilTest:testArgSortDupsEvenItems() (gas: 16387)
+ArrayUtilTest:testArgSortDupsInt() (gas: 15055)
+ArrayUtilTest:testConcat() (gas: 5174)
```

Comment:
@dsshap given that the gas seems to be +40% for these methods, what were the intention behind this change?

Reply:
@antoncoding see above reply